workers/ohsome_quality_analyst/api/api.py

```diff
@@ -161,9 +168,7 @@ async def _fetch_indicator(parameters) -> dict:
         remove_svg_from_properties(geojson_object)
     response = empty_api_response()
     response.update(geojson_object)
-    return JSONResponse(
-        content=jsonable_encoder(response), media_type=MEDIA_TYPE_GEOJSON
-    )
+    return CustomJSONResponse(content=response, media_type=MEDIA_TYPE_GEOJSON)
```

Comment:
doesn't return a `dict` as indicated in the type hints

Reply:
Fixed